int/api/html/front/registry.js

```diff
@@ -24,7 +24,7 @@ function tableInsert(website) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = "http://" + website[NAME] + ".massa/";
+    const url = `//${website[NAME]}.massa:${window.location.port}/`;
```

Comment:
```suggestion
    const port = window.location.port == 80 ? "" : ":" + window.location.port;
    const url = `${windows.location.protocol}//${website[NAME]}.massa${port}/`;
```

Reply:
I'm not sure think we need to add the `protocol` at the beginning since